app/assets/javascripts/discourse/app/components/welcome-topic-banner.js

```diff
@@ -0,0 +1,35 @@
+import { getOwner } from "discourse-common/lib/get-owner";
+import { action } from "@ember/object";
+import Component from "@ember/component";
```

Comment:
Can we use a Glimmer component here, since it's a whole new component?

Reply:
Sure, done.